packages/dashboard/src/app/views/myStories/index.js

```diff
@@ -78,8 +79,16 @@ function MyStories() {
       getAuthors,
     })
   );
-  const { apiCallbacks, canViewDefaultTemplates } = useConfig();
+  const { filtersLoading, getFiltersObject } = useFilters(
+    ({ state: { filtersLoading }, actions: { getFiltersObject } }) => ({
+      filtersLoading,
+      getFiltersObject,
+    })
+  );
+
+  const _filters = JSON.stringify(getFiltersObject());
```

Comment:
What's the reason for using `JSON.stringify` and `JSON.parse()` for the filters? They're not usually cheap calls, so if we can avoid them, that would be nice.

Reply:
This was done because the `filters` in the filters provider was causing an infinite loop as a dependency on the `useEffect` that was calling `fetchStories` but I do see away around that 👍